pkg/bisect/bisect.go

```diff
@@ -646,14 +646,16 @@ func (env *env) test() (*testResult, error) {
 			env.log("reproducer seems to be flaky")
 			env.flaky = true
 		}
-	} else if good != 0 {
+	} else if 2*good > len(results) {
+		// Require at least 50% of successful runs to say it's a good commit.
+		// Otherwise we just have too little data to come to any conclusion.
```

Comment:
Log some readable message as to why we will skip this commit.
It will be confusing in the bisection log otherwise.